products/cloudrun/api.yaml

```diff
@@ -270,6 +265,25 @@ objects:
     required: true
     description: Spec holds the desired state of the Service (from the client).
     properties:
+    - !ruby/object:Api::Type::Array
+      name: traffic
+      description: |-
+        Traffic specifies how to distribute traffic over a collection of Knative Revisions and Configurations
+      item_type: !ruby/object:Api::Type::NestedObject
+        properties:
+        - !ruby/object:Api::Type::String
+          name: revisionName
+          description: |-
+            RevisionName of a specific revision to which to send this portion of traffic.
+        - !ruby/object:Api::Type::Integer
+          name: percent
+          required: true
```

Comment:
AFAIK percent is required to be 100 when using `latestRevision=true` or can be anything when using `revisionName`. If there was any default I think it would be 100 percent, and if they omit the block it will default to using latest revision and 100%. Maybe it would be better to add a `atLeastOneOf` to the other fields to show that a combination of 2 fields is required to set this block?

Reply:
eh its better to require it then if the default changes - i'd just remove the description saying "defaults to 0" or edit it